src/calc/single_param_function.ts

```diff
@@ -91,3 +94,20 @@ const mean = (value: Value): Value => {
 
   return new Value([[(total / count).toString()]]);
 };
+/**
+ * Median of all the cells in the input value, producing a single cell output.
+ */
+const median = (value: Value): Value => {
+  const sortedValues = [...value.val].sort((a, b) => a - b);
+  let isEven;
+  let middleIndex;
+  if (sortedValues.length % 2 == 0) {
+    middleIndex = sortedValues.length / 2;
+    isEven = true;
+  }
+  else {
+    middleIndex = (sortedValues.length - 1) / 2;
+    isEven = false;
+  }
+  return new Value([[(isEven ? (sortedValues[middleIndex - 1] + sortedValues[middleIndex]) / 2 : sortedValues[middleIndex]).toString()]]);
```

Comment:
Can this be refactored to instead have two simpler return statements in the if and else blocks? I think that might make it a bit easier to read.

Reply:
I would agree with this refactoring idea to make the code easier to read.